corehq/apps/reports/forms.py

```diff
@@ -3,7 +3,77 @@
 from corehq.apps.style.forms.fields import MultiEmailField
 from crispy_forms.helper import FormHelper
 from crispy_forms.layout import Submit
-from .models import ReportNotification
+from .models import (
+    ReportConfig,
+    ReportNotification,
+)
+
+
+class SavedReportConfigForm(forms.Form):
+    name = forms.CharField()
+    description = forms.CharField(
+        required=False,
+        widget=forms.Textarea(),
+    )
+    start_date = forms.DateField(
+        required=False,
+    )
+    end_date = forms.DateField(
+        required=False,
+    )
+    date_range = forms.CharField(
+        required=False,
+        widget=forms.HiddenInput(),
+    )
+    days = forms.IntegerField(
+        required=False,
+        widget=forms.HiddenInput(),
+    )
+    _id = forms.CharField(
+        required=False,
+        widget=forms.HiddenInput(),
+    )
```

Comment:
Is this `_id` field possibly used to look up and update an existing record? If so, is there a permissions check to verify that the user is updating a record that they own and not someone elses?